src/bam-fetcher-worker.js

```diff
@@ -404,6 +298,23 @@ const getCoverage = (segmentList, samplingDistance) => {
     });
   }
 
+  // Add thousands separators for easier reading
+  const formatNumber = (num) => String(num).replace(/(.)(?=(\d{3})+$)/g,'$1,');
```

Comment:
Might be better to use something like d3-format which offers more flexibility and legibility.

Reply:
Agreed